rootfs/Dockerfile

```diff
@@ -16,14 +16,16 @@ RUN buildDeps='gcc git libffi-dev libpq-dev python3-dev'; \
         libpq5 \
         python3 \
         sudo && \
-	ln -s /usr/bin/python3 /usr/bin/python && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
```

Comment:
You're right now that I look at the source. 👍  The GH diff display made it look otherwise.